content/zurihac2021/projects.json

```diff
@@ -598,5 +598,17 @@
     },
     "contact": "Li-yao Xia",
     "description": "A web app to visualize Haskell syntax. Written in Purescript."
-  }
+  },
+  {
+  "id": "loggy",
+  "name": "Loggy",
+  "link": "https://github.com/saatvikshah/loggy",
+  "contributor level": {
+    "beginner": true,
+    "intermediate": false,
+    "advanced": false
+  },
+  "contact": "Pedro Jaramillo", "Saatvik Shah"
```

Comment:
okay pretty sure I broke the webpage due to this! - shouldve been "Pedro Jaramillo, Saatvik Shah"

Reply:
Fix is here: https://github.com/zfoh/zfoh.ch/pull/161